packages/core/src/anchor/ethereum/ethereum-anchor-validator.ts

```diff
@@ -6,6 +6,11 @@ import { AnchorProof, AnchorValidator, DiagnosticsLogger } from '@ceramicnetwork
 import { Block, TransactionResponse } from '@ethersproject/providers'
 import { base16 } from 'multiformats/bases/base16'
 import { Interface } from '@ethersproject/abi'
+import { create as createMultihash } from 'multiformats/hashes/digest'
+import * as dagCBOR from '@ipld/dag-cbor'
+import { CID } from 'multiformats/cid'
+
+const SHA256_CODE = 0x12
```

Comment:
```suggestion
import { CID } from 'multiformats/cid'

const DAG_CBOR_CODE = 0x71
const SHA256_CODE = 0x12
```
We don't really need the dagcbor dep here.

source: https://github.com/multiformats/multicodec/blob/master/table.csv#L44